nextroute/model_expression_custom.go

```diff
@@ -124,10 +137,16 @@ type StopExpression interface {
 	)
 }
 
+// VehicleExpression is an expression that implements a DefaultExpression and is
```

Comment:
We do not require a VehicleExpression, we can have the VehicleTypeDistanceExpression be a VehicleTypeExpression.